pkg/exchange/bitget/exchange.go

```diff
@@ -383,9 +385,53 @@ func (e *Exchange) QueryClosedOrders(ctx context.Context, symbol string, since,
 	return types.SortOrdersAscending(orders), nil
 }
 
-func (e *Exchange) CancelOrders(ctx context.Context, orders ...types.Order) error {
-	// TODO implement me
-	panic("implement me")
+func (e *Exchange) CancelOrders(ctx context.Context, orders ...types.Order) (errs error) {
+	if len(orders) == 0 {
+		return nil
+	}
+
+	for _, order := range orders {
+		req := e.client.NewCancelOrderRequest()
+
+		reqId := ""
+		switch {
```

Comment:
switch case works, but why not use if else clause?

Reply:
just i don't like the nested if-else statements